components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -380,13 +380,18 @@ void SHOCMacrophysics::initialize_impl (const RunType run_type)
                                  T_mid, dse, z_mid, phis);
 
   // Set field property checks for the fields in this process
-  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("T_mid"),m_grid,140.0,500.0,false);
-  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("qv"),m_grid,1e-13,0.2,true);  // This is a quick fix to make sure qv doesn't go negative.  TODO item is to take care of this in a more clever way.
-  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("qc"),m_grid,0.0,0.1,false);
-  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("horiz_winds"),m_grid,-400.0,400.0,false);
-  add_postcondition_check<FieldPositivityCheck>(get_field_out("pbl_height"),m_grid);
-  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("cldfrac_liq"),m_grid,0.0,1.0,false);
-  add_postcondition_check<FieldPositivityCheck>(get_field_out("tke"),m_grid);
+  auto eps = std::numeric_limits<Real>::epsilon();
+  using FWIC = FieldWithinIntervalCheck;
```

Comment:
Not a big deal, but maybe `WithinInterval` and `LowerBound` might be more eye-catching than `FWIC` and `FLBC` in the lines that follow?